src/okteto.ts

```diff
@@ -144,7 +144,14 @@ export function start(manifest: string, namespace: string, name: string, port: n
     }
   });
 
-  term.sendText(`${getBinary()} up -f ${manifest} --remote ${port}`, true);
+  
+  var binary = getBinary();
```

Comment:
block-scoped `let` is recommended instead of `var`.

```suggestion
  let binary = getBinary();
```